Demo/Common/IntegrationExample.swift

```diff
@@ -53,15 +55,19 @@ internal final class IntegrationExample: APIClientAware {
 
     // MARK: - Networking
 
-    internal func requestPaymentMethods() {
-        let request = PaymentMethodsRequest()
+    internal func requestPaymentMethods(order: PartialPaymentOrder? = nil,
+                                        completion: ((PaymentMethods) -> Void)? = nil) {
+        let request = PaymentMethodsRequest(order: order)
```

Comment:
what if order is nil?

Reply:
that's fine `PaymentMethodsRequest.order` is optional